presidio-anonymizer/Dockerfile.dev

```
@@ -1,7 +1,7 @@
 # Dockerfile
-FROM python:3.13-slim
+FROM python:3.13-slim@sha256:3de9a8d7aedbb7984dc18f2dff178a7850f16c1ae7c34ba9d7ecc23d0755e35f
 
 RUN apt-get update \
   && apt-get install -y build-essential
 
-RUN pip install poetry
+RUN pip install poetry==2.3.2
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)